scripts/issues.coffee

```diff
@@ -43,19 +45,24 @@ module.exports = (robot) ->
           if counter is issue_index
             msg.reply "What about #{obj.title}? Take a look at it here: #{obj.html_url} !"
 
-  robot.hear /#(?:(?:(\w+)\/)?(\w+)\/)?(\d+)/i, (msg) ->
-    issueNo = msg.match[3]
-    repositoryFullName = ""
-    organizaton = "MovingBlocks"
-    repository = "Terasology"
-    if (msg.match[2]?)
-      repository = msg.match[2]
-      organizaton = "Terasology"
-      if (msg.match[1]?)
-        organizaton = msg.match[1]
-    repositoryFullName = "#{organizaton}/#{repository}"
-    robot.logger.info "Querying https://api.github.com/repos/#{repositoryFullName}/issues/#{issueNo}"
-    robot.http("https://api.github.com/repos/#{repositoryFullName}/issues/#{issueNo}")
+  robot.hear /#(?:(?:(\w+)\/)?(\w+)\/)?(\d+)/gi, (msg) ->
+    num = 0
+    while msg.match[num]?
+      word = msg.match[num]
```

Comment:
It would be good if you made it clearer what `word` refers to in the variable name. Replace it with `issue` maybe?